cli/admin/src/main/java/io/pravega/cli/admin/controller/ControllerCommand.java

```diff
@@ -50,7 +61,39 @@ protected Context createContext() {
         ClientConfig clientConfig = new ClientConfig();
         clientConfig.register(JacksonJsonProvider.class);
         clientConfig.property("sun.net.http.allowRestrictedHeaders", "true");
-        Client client = ClientBuilder.newClient(clientConfig);
+
+        Client client;
+
+        // If tls parameters are configured, set them in client
+        if (getCLIControllerConfig().isTlsEnabled()) {
+            KeyStore ks = null;
+            InputStream trustStore = null;
+            try {
+                trustStore = new FileInputStream(new File(getCLIControllerConfig().getTruststore()));
```

Comment:
I suggest using try-with-resources or annotating with with `@Cleanup` to ensure this is closed. If you get an error between here and line 75, your input stream will not be closed.

Reply:
Used `@Cleanup` on the `InputStream` as suggested.